pkgs/by-name/hu/hugo/package.nix

```diff
@@ -12,22 +12,25 @@
 
 buildGoModule rec {
   pname = "hugo";
-  version = "0.136.5";
+  version = "0.138.0";
 
   src = fetchFromGitHub {
     owner = "gohugoio";
     repo = "hugo";
     rev = "refs/tags/v${version}";
-    hash = "sha256-NluV9/tqWTRfs2u9g4cHC1TyrxOXx3ghOZ4GDx7OZFo=";
+    hash = "sha256-IDWQRPJrTCkvcTcsaGuyQraVoWWUe0d6FTQvvYHZcD0=";
   };
 
-  vendorHash = "sha256-KqDsa7MlSONyn7AYOepQ95q1CEM83AhWk23iYSQ4twU=";
+  vendorHash = "sha256-5YS76L7kisyPz8yv2RCgZHpY/AkjdHE+SUwMOuo3uLg=";
 
   doCheck = false;
 
   proxyVendor = true;
 
-  tags = [ "extended" ];
+  tags = [
+    "extended"
+    "withdeploy"
```

Comment:
Why this change?

Reply:
Hugo v0.137 put `hugo deploy` behind that build tag, so I added it in case anyone is using that feature nowadays.